Modules/System/Email/src/Message/EmailMessageImpl.Codeunit.al

```diff
@@ -355,6 +355,36 @@ codeunit 8905 "Email Message Impl."
         end;
     end;
 
+    procedure AddRecipient(RecipientType: Enum "Email Recipient Type"; Recipient: Text)
+    var
+        EmailRecipientRecord: Record "Email Recipient";
+        UniqueRecipients: Dictionary of [Text, Text];
+        Recipients: List of [Text];
+        tmpRecipient: Text;
+    begin
+        if Recipient <> '' then
```

Comment:
This means that you do not add anything if the recipient is not empty.
Is that on purpose?

Reply:
Oops, I meant of course to exit if the recipient is empty, not if it is not. Good Catch!